neighborhoodwatch/cu_knn.py

```diff
@@ -208,11 +212,12 @@ def process_batches(final_indecies_filename,
                     split,
                     engine='raft'
                     ):
-    try:
-        os.remove(final_indecies_filename.replace(".parquet", f"*.parquet"))
-        os.remove(final_distances_filename.replace(".parquet", f"*.parquet"))
-    except OSError:
-        pass
+    # try:
+    #     os.remove(final_indecies_filename.replace(".parquet", f"*.parquet"))
+    #     os.remove(final_distances_filename.replace(".parquet", f"*.parquet"))
+    # except OSError:
+    #     pass
+
```

Comment:
do you need these files

Reply:
That was temporarily needed and fixed now.